apps/website/blog/2024-12-01-maci-getting-started.md

```diff
@@ -0,0 +1,294 @@
+
+```bash
+pnpm download-zkeys:ceremony
+```
+
+:::info
+Currently, the ceremony artifacts work with MACI version up to 2.x
+:::
+
+## Deploy Contracts
+
+Before deploying the contracts we need to do some final configurations to our repository.
+
+### Set the environment variables
+
+Head to the `packages/contracts` folder and copy the `.env.example` file.
+
+```bash
+cd packages/contracts && \
+cp .env.example .env
+```
+
+Make sure to include a mnemonic and RPC url (make sure to replace NETWORK with the network you want to use).
+
+```js
+MNEMONIC = "your ethereum secret key";
+NETWORK_RPC_URL = "the eth provider url";
+NETWORK_ETHERSCAN_API_KEY = "etherscan api key";
+```
+
+### Generate Coordinator Keys
+
+In order to run MACI polls, a coordinator is required to publish their MACI public key. You will need to generate a MACI keypair, and treat the private key just as your ethereum private keys. Please store them in a safe place as you won't be able to finish a round if you lose access, or if compromised a bad actor could decrypt the vote and publish them online. You can generate a new key pair using maci-cli by running the following commands:
+
+```bash
+cd packages/cli && \
+node build/ts/index.js genMaciKeyPair
+```
+
+### Set the configuration file
+
+:::note
+There are already some deployed contracts that could be reused. More information can be found in the (incremental documentation page)[docs/getting-started#deploy-maci-contracts].
+:::
+
+Head back to the contracts folder and copy the config example and update the fields as necessary:
+
+```bash
+cd ../contracts && \
+cp deploy-config-example.json deploy-config.json
+```
+
+#### ConstantInitialVoiceCreditProxy
+
+Defines how many credits will get each voter.
+
+#### Gatekeeper
+
+MACI uses a "gatekeeper" contract to configure and enforce the eligibility criteria of voters who can participate in MACI polls. In other words, it is a way to allowlist signups to the system to protect against sybil attacks. Please refer to the [gatekeeper page in the documentation](/docs/technical-references/smart-contracts/Gatekeepers) for more information on the supported Gatekeepers.
+
+:::important
+For testing we suggest using the **FreeForAllGatekeeper** as it allows anyone to signup on MACI.
+:::
+
+#### MACI
+
+This property defines which Gatekeeper and stateTreeDepth MACI is going to use.
+
+:::important
+The stateTreeDepth value for test artifacts is: **10**. For ceremony keys: **14**.
+:::
+
+#### VkRegistry
+
+Here we define the path to the zero knowledge artifacts we downloaded in the previous steps and the Verifying key registry used for generating proofs.
+
+:::important
+The values for test keys are: **10-1-2-2-1**. For ceremony keys: **14-5-9-3-2**.
+:::
+
+#### Poll
+
+Sets the poll duration in seconds, if we use quadratic voting or not and the public key for the Coordinator.
```

Comment:
suggest "Configures the poll duration in seconds, determines whether quadratic voting is enabled, and sets the public key of the Coordinator."